[2025-04-09T07:11:16.625+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.extract_spotify manual__2025-04-09T07:11:12.014769+00:00 [queued]>
[2025-04-09T07:11:16.634+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.extract_spotify manual__2025-04-09T07:11:12.014769+00:00 [queued]>
[2025-04-09T07:11:16.635+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-09T07:11:16.755+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): extract_spotify> on 2025-04-09 07:11:12.014769+00:00
[2025-04-09T07:11:16.763+0000] {standard_task_runner.py:60} INFO - Started process 94 to run task
[2025-04-09T07:11:16.767+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline_modo_prueba', 'extract_spotify', 'manual__2025-04-09T07:11:12.014769+00:00', '--job-id', '78', '--raw', '--subdir', 'DAGS_FOLDER/etl_workflow.py', '--cfg-path', '/tmp/tmptk38b14_']
[2025-04-09T07:11:16.770+0000] {standard_task_runner.py:88} INFO - Job 78: Subtask extract_spotify
[2025-04-09T07:11:16.885+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline_modo_prueba.extract_spotify manual__2025-04-09T07:11:12.014769+00:00 [running]> on host 6d74a53745cf
[2025-04-09T07:11:17.134+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='simon' AIRFLOW_CTX_DAG_ID='etl_pipeline_modo_prueba' AIRFLOW_CTX_TASK_ID='extract_spotify' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T07:11:12.014769+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T07:11:12.014769+00:00'
[2025-04-09T07:11:17.137+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-09T07:11:17.138+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/scripts/extract_spotify.py']
[2025-04-09T07:11:17.154+0000] {subprocess.py:86} INFO - Output:
[2025-04-09T07:11:17.711+0000] {subprocess.py:93} INFO - error al extraer Spotify: [Errno 2] No such file or directory: '../data/raw/spotify/spotify.csv'
[2025-04-09T07:11:17.822+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-09T07:11:17.894+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline_modo_prueba, task_id=extract_spotify, execution_date=20250409T071112, start_date=20250409T071116, end_date=20250409T071117
[2025-04-09T07:11:17.983+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-09T07:11:18.011+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
